wkhtmltopdf-image/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+################
+# Docker file to build a python container hosting the wkhtmltopdf service.
+################
+
+#Based off of the ubuntu container
+FROM ubuntu:xenial
```

Comment:
I really, really wanted to use Alpine, but I'd have to compile/build wkhtmltopdf from source and mess with xvfb...  This leaves us with a 750MB container image... but that's the same size as the one we were using before from OpenLabs so we haven't lost a whole lot.